stable_baselines3/common/base_class.py

```diff
@@ -399,6 +399,9 @@ def _setup_learn(
         :return:
         """
         self.start_time = time.time()
+
+        self.policy.train()
```

Comment:
not sure to understand why is it needed there.
I would expect `.train()` to be called before taking gradient steps and `.eval()` after taking them